docs/simple_examples.rst

```diff
@@ -0,0 +1,148 @@
+******************************
+Simple scintillometry examples
+******************************
+
+Consider observing a pulsar using a single radio dish.
+Let's assume that the pular is stationary w.r.t. Earth,
```

Comment:
I'd do "Let's first assume " and omit "that its spectrum is flat, OTHER" - or, rather, move them to the end in the notes section, adding a sentence about "implicit assumptions".

Reply:
Done